src/sentry/static/sentry/app/views/organizationDashboard/widgetChart.jsx

```diff
@@ -65,6 +73,29 @@ class WidgetChart extends React.Component {
       ...(isTable && {
         headerProps: {hasButtons: true},
         extraTitle: <ExploreWidget {...{widget, router, selection}} />,
+        rowClassName: tableRowCss,
+        getRowLink: obj => {
+          if (!obj || typeof obj.name === 'undefined') return null;
+
+          // Table Charts don't support multiple queries
+          const [query] = widget.queries.discover;
+
+          return getEventsUrlPathFromDiscoverQuery({
+            organization,
+            selection,
+            query: {
+              ...query,
+              conditions: [
+                ...query.conditions,
+                ...zipWith(query.groupby, obj.name.split(','), (field, value) => [
```

Comment:
I'm a bit lost with these conditions - what is `obj` here supposed to be anyway? It might be worthwhile extracting this into a utility function with some comments.

Reply:
Refactored and added some tests https://github.com/getsentry/sentry/pull/11744/commits/7af50c5ca8ca8926c3a333d5c254e98e71e9949b